docs/ext/commands/slash_commands.rst

```diff
@@ -142,7 +142,7 @@ All the other types may be converted implicitly, similarly to :ref:`ext_commands
         ...
 
 .. note::
-  \* All channel subclasses and unions are also supported. See :attr:`.ParamInfo.channel_types` for more fine-grained control
+  \* All channel subclasses and unions are also supported. See :attr:`ParamInfo.channel_types <ext.commands.ParamInfo>` for more fine-grained control
```

Comment:
missing channel_types

Reply:
Unfortunately can't link to it since it's documented in `Parameters` now, not `Attributes`

Reply:
Understandable